### PR TITLE
Updated Readme

### PR DESCRIPTION
**Pull request by ayush-sleeping**
## Related Issue or bug
  - Updated Readme

Fixes: #[ Readme Update #63 ]

#### Describe the changes you've made
A clear and concise description of what you have done to successfully close your assigned issue. Any new files? or anything you feel to let us know!

## How Has This Been Tested?

For Final Output Visit here : https://github.com/ayush-sleeping/classDeck/blob/develop/README.md

## Checklist:

- [x] Have you checked there aren't other open Pull Requests for the same update/change?
- [x] My submission doesn't break any existing feature.
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests pass locally with my changes.
- [x] Any dependent changes have been merged and published in downstream modules.

# 
# 

Hey maintainer , Checkout this PR and if all things are good to go , Kindly merge it ..........

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,108 @@
...
+![Repo. Size](https://img.shields.io/github/repo-size/coding-geek21/classDeck?color=white) 
+
+</div>
+
+<!-- ---------------------------------------------------------------------------------------------------------------------- -->
+
+<!-- ---------------------------------------------------------------------------------------------------------------------- -->
+
+<p align="center">
+    <a href="https://classdeck.herokuapp.com/"><strong>ZeroOctave Official site »</strong></a>
```
Comment (coding-geek21) [inline README.md]: Change it to ClassDeck Official site

File: README.md [hunk 2]
```diff
@@ -1,33 +1,108 @@
-# ClassDeck
+<div id="top"></div>
 
-## Introduction 📜
 
+<h1 align="center"> ClassDeck </h1>
```
Comment (coding-geek21) [inline README.md]: Can you add the ClassDeck logo before the heading?
Link :  https://github.com/coding-geek21/classDeck/blob/develop/classDeck/static/img/logo.png

**Review by coding-geek21 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ayush-sleeping) [reply README.md]: I am in College now , When I will reach home in the evening, I will complete this task....

**Comment by ayush-sleeping** [comment]
hey @coding-geek21 , Checkout now, 
 https://github.com/ayush-sleeping/classDeck/blob/develop/README.md

**Pull request merged**